### ISSUE TITLE
Get-RemoteProgram - Why filter the creation of the output object

### ISSUE THREAD

**Issue by AspenForester** [jaapbrasser/SharedScripts#17]
I'm curious as to why you chose to do the following when creating the object to output
```
New-Object -TypeName PSCustomObject -Property $HashProperty |
                                Select-Object -Property $SelectProperty
```
instead of just
```
New-Object -TypeName PSCustomObject -Property $HashProperty
```

**Comment by jaapbrasser** [comment]
Because otherwise the properties of the object come out in the order dictated by the hashtable. As this code is written with PowerShell 2.0 compatibility in mind this is the workaround I used in order to display the properties in the order I intended.

It is not a very elegant solution...

**Comment by jaapbrasser** [comment]
Fixed by e89cf414a3403344ab40dec2120ef14392edcaa8, In PowerShell 3 and up this notation is no longer used.
